templates/idp/email-password-interaction/resetPassword.ejs

```diff
@@ -13,7 +24,7 @@
   </header>
   <main>
     <h1>Reset password</h1>
-    <form action="/idp/resetpassword" method="post">
+    <form id="form" action="/idp/resetpassword" method="post">
```

Comment:
Shall we just leave the `action` off, such that we don't have to perform the "URL stays the same" trick?

Reply:
That would actually be the best solution, since then I could use a URL like `/idp/resetpassword/1234` instead of using a query parameter and no frontend javascript would be needed. But when I looked into this I found conflicting information on if this was allowed or not. If yes then I'm going for that solution indeed.

Reply:
> conflicting information

Authoritative source says it's optional:

> > The attributes for form submission that may be specified on form elements are action, enctype, method, novalidate, and target.

—https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#attr-fs-action